### DISCUSSION TITLE
Run Pure FTP service as non root

### DISCUSSION THREAD

**Discussion by Deepak-Suresh** [stilliard/docker-pure-ftpd#171] [General]
Hi,

Please add steps to run the pure ftp service as non root user..

this may be required if the kubenetes cluster has some Pod Security policies like "mustRusAsNonRoot" .. So no containers can run the services inside as root user..

I saw an option in pure ftp documentation like "--with-nonroot" to run the service as non-root user.

any help on this is much appreciated!!

**Comment by mpezzull** [comment]
I have to do the same thing but i don't know if this image is good or i have to check somewhere else, can you tell me what you've done? Thank you very much
